device/dell/x86_64-dellemc_z9432f_c3758-r0/plugins/psuutil.py

```diff
@@ -137,15 +148,18 @@ def get_psu_status(self, index):
         psu_status = '0'
         
         if index == 1:
-           cmd_status, psu_status = commands.getstatusoutput(self.IPMI_PSU1_DATA)
+            p = subprocess.run(self.IPMI_PSU1_DATA, capture_output=True, universal_newlines=True)
+            cmd_status, line = p.returncode, p.stdout.replace('\n', '')
         elif index == 2:
-           cmd_status, psu_status = commands.getstatusoutput(self.IPMI_PSU2_DATA)
+            p = subprocess.run(self.IPMI_PSU2_DATA, capture_output=True, universal_newlines=True)
+            cmd_status, line = p.returncode, p.stdout.replace('\n', '')
         else:
            logging.error("Invalid PSU number:" + index)
-  
+        psu_status = line[8] if len(line) > 8 else '0'
```

Comment:
>line [](http://example.com/codeflow?start=36&length=4)

Is `line` a string or an array of string?

Reply:
looking at original behavior, I believe `line` is a string.